modules/PrimaryPool/substreams.yaml

```diff
@@ -27,13 +34,27 @@ modules:
     kind: map
     initialBlock: 8586768
     inputs:
+      - source: sf.ethereum.type.v2.Block
+      # Added UP
       - map: map_pools
     output:
-      type: proto:verified.primary.v1.Subscriptions
+      type: proto:verified.primary.v1.PrimaryIssues
+
+  - name: map_subscriptionsCheck
```

Comment:
@yaksheshgupta are we using map_subscriptions or map_subscriptionscheck ? One seems to be taking map pools as input - is this what you were mentioning yesterday on telegram ?

Reply:
Yes "map_subscriptionscheck" is just a function that return "subscriptions" event. Similar to "map_pools".
Its just for testing purposes